tests/conftest.py

```diff
@@ -82,8 +82,8 @@ def TEST_ADLS_FILE_PATH_CSV():
 
 
 @pytest.fixture(scope="session", autouse=True)
-def s3_config_key():
-    return os.environ.get("S3_CONFIG_KEY")
+def aws_config_key():
+    return os.environ.get("AWS_CONFIG_KEY")
```

Comment:
Pls then also modify relevant `.env.example` (the one in `tests/`?)

Reply:
[🚚 Rename S3_CONFIG_KEY env var to AWS_CONFIG_KEY](https://github.com/dyvenia/viadot/pull/636/commits/7719be2cae187395bf54db858fff32e504c387ea)